docs/src/Box.doc.js

```diff
@@ -419,11 +419,54 @@ card(
       />
     </MainSection.Subsection>
     <MainSection.Subsection
-      title="Using role"
+      title="Using 'as' property"
       description={`
-    Setting the  \`role\` property on Box classifies the Box as the semantically appropriate HTML element. For example, setting \`role="banner"\` will designate that Box to be the equivalent of a \`<header>\` within the page, allowing assistive technology to classify the Box appropriately.
+      By default, the Box component renders a \`div\` element, which doesn't provide much meaning to the user or assistive technology. Use the \`as\` prop to inform which HTML component should be rendered by the Box component to ensure a meaningful experience.
 
-    Using the \`role\` property can prevent an excess of generic \`<div>s\` and give the user better context. Learn more about [ARIA roles](https://www.w3.org/TR/wai-aria/#usage_intro).
+      When using a Box component as a custom element, it is your responsibility to address all the accessibility implications. Both the \`role\` and \`as\` properties semantically classify the Box; however, the \`as\` prop defines a more concise way to describe the HTML element by modifying the underlying DOM element directly, which helps support both accessibility and SEO. We recommend using the \`as\` prop when possible, making sure that the prop type is semantically associated with the Box content.
```

Comment:
"We recommend using" > "Use the \`as\` prop when possible, making sure that the prop type is semantically associated with the Box content."

Reply:
Hahaha, I like the "we recommend", it makes it a bit friendly.